src/main/kotlin/ua/marchenko/artauction/common/mongock/migration/ArtAuctionMigrationChangeUnit.kt

```diff
@@ -0,0 +1,41 @@
+package ua.marchenko.artauction.common.mongock.migration
+
+import io.mongock.api.annotations.ChangeUnit
+import io.mongock.api.annotations.Execution
+import io.mongock.api.annotations.RollbackExecution
+import org.slf4j.LoggerFactory
+import org.springframework.data.domain.Sort
+import org.springframework.data.mongodb.core.MongoTemplate
+import org.springframework.data.mongodb.core.index.Index
+import org.springframework.data.mongodb.core.index.IndexOperations
+import ua.marchenko.artauction.user.model.MongoUser
+
+@ChangeUnit(id = "ArtAuctionMigrationChangeUnit", order = "001", author = "svitlana marchenko", systemVersion = "1")
+class ArtAuctionMigrationChangeUnit {
```

Comment:
Why migration is called ArtAuctionMigrationChangeUnit but you create index for user inside. Mirror migration action to its name